### DISCUSSION TITLE
docker exec influxdb influx apply -f /var/lib/influxdb2/influx_dashboard.json --org vuegraf --force yes -t <my-influx-token> fails

### DISCUSSION THREAD

**Discussion by 2006flhtcuse** [jertel/vuegraf#170] [General]
Trying to import the dashboard with d"ocker exec influxdb influx apply -f /var/lib/influxdb2/influx_dashboard.json --org vuegraf --force yes -t <my-influx-token>" with the correct token inserted but it fails with:
Error: failed to read template(s) from "/var/lib/influxdb2/influx_dashboard.json": invalid character '<' looking for beginning of value

**Comment by jertel** [comment]
The dashboard JSON file may be corrupted. How did you download it?

**Comment by 2006flhtcuse** [comment]
wget into the linux platform

**Comment by jertel** [comment]
Which URL did you wget? If you fetched the wrong link it could have HTML embedded around the JSON content.

**Comment by 2006flhtcuse** [comment]
wget https://github.com/jertel/vuegraf/blob/master/influx_dashboard.json

**Comment by jertel** [comment]
Use this link: 
```
wget https://raw.githubusercontent.com/jertel/vuegraf/refs/heads/master/influx_dashboard.json
```

**Comment by 2006flhtcuse** [comment]
That fixed it!! thank you!

**Comment by 2006flhtcuse** [comment]
I have this in the empora web 
![image](https://github.com/user-attachments/assets/8013e6c9-fc8e-443c-b97b-644de9235abe)
I have this in the config
![image](https://github.com/user-attachments/assets/8d4896c3-a8d9-46e7-9637-4b3cbf69babd)
But I only see this in teh dashboard
![image](https://github.com/user-attachments/assets/bedc7534-a703-43c0-9472-ce925c486eeb)

Not sure where I am going wrong here.

**Comment by 2006flhtcuse** [comment]
email/password blanked with 88888888 for the forum

**Reply by jertel** [reply]
Add the verbose `--verbose` or debug `--debug` option to the command line (as described in the instructions) and let's see what the log has to say. Also, include a full dashboard screenshot showing the dropdowns.
